bin/fetch

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+VERSION=1.13
+URL="http://download.ckeditor.com/codemirror/releases/codemirror_$VERSION.zip"
+
+if [ -L "$0" ]; then
+  echo Executing this script as a symlink is not supported. 2>&1
+  exit 1
+fi
+
+echo Check if the required executables are available.
+which unzip curl mktemp
+
+cd "$(dirname "$0")"
+cd ..
+
+echo Create a temporary directory for extraction.
+mkdir -p tmp
+cd tmp
+
+TMP_FILE=$(mktemp)
+curl -o"$TMP_FILE" "$URL"
+unzip "$TMP_FILE"
+rm -v "$TMP_FILE"
+
+echo Remove the unnecessary execute bit of the distributed files.
+find . -type f -exec chmod -x {} ';'
```

Comment:
Isn't this unnecessary?

And if we intend this, then why on the current dir instead of the unzipped result?